### DISCUSSION TITLE
Qt components

### DISCUSSION THREAD

**Discussion by Marin-Kitagawa** [qutebrowser/qutebrowser#6604] [Q&A] [answered]
None of the components that are listed under `Qt` (see below) are available in the Custom Installation of the latest Qt open source installer version `6.1.1` (there are other versions as well). Could you please say what components must be selected in the custom installation of the Qt installer? There are no components by any of the names given below. I am new to this. Could you please say what components must be selected?

```
Qt 5.12.0 or newer (5.12 LTS or 5.15 recommended) with the following modules:

QtCore / qtbase

QtQuick (part of qtbase or qtdeclarative in some distributions)

QtSQL (part of qtbase in some distributions)

QtDBus (part of qtbase in some distributions; note that a connection to DBus at runtime is optional)

QtOpenGL

QtWebEngine, or

alternatively QtWebKit (5.212)
```

**Comment by toofar** [comment, marked as answer]
QtWebengine isn't in Qt6 until the upcoming 6.2 release: https://www.qt.io/blog/add-on-support-in-qt-6.0-and-beyond

But what is this "Qt open source installer" with its "Custom Installation" feature? Assuming it is the green thing shown in this video https://www.youtube.com/watch?v=wPcVlP_wrew you need to install 5.15 for now.

**Reply by Marin-Kitagawa** [reply]
Yes, it is. This is what I meant by the terms above

### Open source Installation
![image](https://user-images.githubusercontent.com/49131888/126028438-1b366c6c-52bc-4f56-9f15-d76062728243.png)
![image](https://user-images.githubusercontent.com/49131888/126028489-7f8263a6-cdff-49e8-8d18-a526a44bdfc4.png)



### Custom Installation
![image](https://user-images.githubusercontent.com/49131888/126028531-893442ab-be87-4f17-b0b6-2cfd4c585d68.png)


### Packages available for 6.1.2
![image](https://user-images.githubusercontent.com/49131888/126028587-aeaac568-c147-422c-93fd-06246fc39f6f.png)

### Packages available for 5.15.x
![image](https://user-images.githubusercontent.com/49131888/126028612-a0d74fa8-dcd5-4d85-ae08-29e9312616da.png)

### Additional libraries
![image](https://user-images.githubusercontent.com/49131888/126028627-c92dc310-5f8c-42da-aded-3a20b95d90e5.png)

### Developer and Design Tools
![image](https://user-images.githubusercontent.com/49131888/126028639-7154f934-41de-45df-ba24-3b9c8ceb01a9.png)


None contains any of the package listed in the `README.md` (as listed in the issue above)

**Reply by toofar** [reply]
I'm pretty sure out of those ones listed under 5.15.2 and Additional libraries only "Qt Webengine" is required, all the rest must be bundled into the core install. I would probably check Qt 3D and Qt Quick 3D for good luck but I don't think it needs them.
